Yoast/wordpress-seo
# Add env to grunt

## Summary

This PR can be summarized in the following changelog entry:

* [non-user-facing] Environment variables added to developer dependencies.
* [non-user-facing] Automatic Slack Notification added to the new release automatisation.

## Relevant technical choices:

* Partially hijacked issue to also set Slack Token in the new .env and use it to finish the notify-slack task.

## Test instructions

This PR can be tested by following these steps:

* Run `grunt notify-slack:rc` in your terminal. Notice that it will send a colorful, well, notice.
* Now copy the `env.example` to  a `.env` file. Note that the `.env` file should **not** be picked up by git, because of the `.gitignore` settings.
* Enter a value behind the `SLACK_TEST_CHANNEL_TOKEN`, and you should still get the message (because it is not a real valid Slack URL.
* You could DM me for a valid token, if you want to see it actually work.


## UI changes

## Documentation

## Quality assurance


Fixes #14044
